src/createAdder.js

```diff
@@ -22,7 +22,18 @@
  * @return {number}
  */
 function createAdder(initialValue = 0) {
-  // write code here
+  let sum = initialValue;
+
+  return (...arg) => {
+    let currentSum = 0;
+    if (arg.length > 0) {
```

Comment:
rewrite without `if` and using features of `.reduce()`

Reply:
Fixed!